content/v2.0/reference/api/influxdb-1x/query.md

```diff
@@ -0,0 +1,115 @@
+---
+title: /query 1.x compatibility API
+list_title: /query
+description: >
+  The `/query` 1.x compatibility endpoint queries InfluxDB 2.0 using **InfluxQL**.
+menu:
+  v2_0_ref:
+    name: /query
+    parent: 1.x compatibility
+weight: 301
+v2.0/tags: [influxql, query]
+products: [cloud]
+list_code_example: |
+  <pre>
+  <span class="api get">GET</span> http://localhost:9999/query
+  </pre>
+related:
+  - /v2.0/query-data/influxql
+---
+
+The `/query` 1.x compatibility endpoint queries InfluxDB 2.0 using **InfluxQL**.
+Use the `GET` request method to query data from the `/query` endpoint.
+
+<pre>
+<span class="api get">GET</span> http://localhost:9999/query
+</pre>
+
```

Comment:
I would re-add the NOTE from above here...  If you have existing buckets that don't follow the **database/retention-policy** naming convention AND were not created via the 1.x /write compatibility API, you **MUST** first create a database and retention policy mapping for these buckets in order for you to query them via the compatibility API.

Use the /api/v2/dbrps API endpoint to
manually create and manage DBRP mappings.  (provide links to the location of these instructions)